### PR TITLE
fix channel number for amsr2

### LINKED ISSUE #1332: amsr2 converter updates
## Current behavior (describe the bug)

testing end-to-end through skylab found some small cleanup to be done

the channel numbers started with 0 rather than 1
had to double check the order of the horizontal and vertical polarization channels
various scaling factors be applied correctly for factors
ensure satellite and solar angles are computed correctly

### PR DESCRIPTION
**Pull request by BenjaminRuston**
## Description

the channel number should begin with 1
some scaling factors need to be applied
the solar zenith needs to be computed
channel ordering 

## Issue(s) addressed

Resolves #1332

### CODE REVIEW

**Comment by BenjaminRuston** [comment]
@smaticka this is the small issue I noted leaving this as a draft until we complete the testing through the minimization

**Review by smaticka — APPROVED** [review]
I was able to run both `hofx3d-c384` and `skylab-atm-land` (with `monitoring` on) with the converted data. 

Thanks @BenjaminRuston !

**Review by PatNichols — APPROVED** [review]
Look good

**Comment by PatNichols** [comment]
@BenjaminRuston Looks like this is good to go except for coding norms failures.

**Comment by BenjaminRuston** [comment]
@smaticka, @PatNichols and @fcvdb have verified the longitudes in the raw files are -180 - 180 following convention

File: src/hdf5/amsr2_2ioda.py [hunk 1]
```diff
@@ -147,11 +147,11 @@ def get_data(f, obs_data):
...
-    obs_data[('sensorChannelNumber', metaDataName)] = np.array(np.arange(nchans), dtype='int32')
+    obs_data[('sensorChannelNumber', metaDataName)] = np.array(np.arange(nchans)+1, dtype='int32')
     k = 'sensorScanPosition'
     obs_data[(k, metaDataName)] = np.tile(np.arange(nbeam_pos, dtype='int32')+1, (nscans, 1)).flatten()
     k = 'sensorZenithAngle'   # ~55.2 incidence angle
-    obs_data[(k, metaDataName)] = np.array(f['Earth Incidence'], dtype='float32').flatten()
+    obs_data[(k, metaDataName)] = np.array(f['Earth Incidence']*f['Earth Incidence'].attrs['SCALE FACTOR'][0], dtype='float32').flatten()
     instr_scan_ang = obs_data[(k, metaDataName)]
     k = 'sensorAzimuthAngle'
     obs_data[(k, metaDataName)] = np.array(f['Earth Azimuth'], dtype='float32').flatten()
```
Comment (fabiolrdiniz) [inline src/hdf5/amsr2_2ioda.py:157]:
```suggestion
    obs_data[(k, metaDataName)] = np.array(f['Earth Azimuth']*f['Earth Azimuth'].attrs['SCALE FACTOR'][0], dtype='float32').flatten()
```

**Review by fabiolrdiniz — CHANGES_REQUESTED** [review]
Thanks, @BenjaminRuston! The same needs to be done for `solarAzimuthAngle` & `solarZenithAngle` (commented out).

**Review by fabiolrdiniz — APPROVED** [review]
Thanks, @BenjaminRuston!

**Pull request merged**